quickemu

```diff
@@ -0,0 +1,47 @@
+---
+tags: [ virtualization ]
+---
+# To run a vm
```

Comment:
```suggestion
# To run a specific vm
```

Optional change: `a specific` may sound better.

```suggestion
# To run a vm:
```

Optional change: trailing colon can be added.

```suggestion
# Run a vm
```

Optional change: `to` can be removed.

```suggestion
# Run a specific vm:
```

Optional change: fixes can be applied together.